elements/bulbs-video/components/revealed.js

```diff
@@ -247,7 +252,7 @@ export default class Revealed extends React.Component {
 
     player.setup(playerOptions);
 
-    GoogleAnalytics.init(player, videoMeta.gaPrefix);
+    GoogleAnalytics.init(player, videoMeta.sendAnalyticsEvent);
```

Comment:
Not sure how I feel about passing a function by reference like this @collin / @MelizzaP.  Seems like the `sendAnalyticsEvent` logic using the GA extraction should just live in the GoogleAnalytics plugin.